rafthttp/transport.go

```diff
@@ -110,10 +116,23 @@ type Transport struct {
 	prober probing.Prober
 }
 
-func (t *Transport) Start() {
+func (t *Transport) Start() error {
+	var err error
+	// Read/write timeout is set for stream roundTripper to promptly
+	// find out broken stream connection. If it fails to detect broken
```

Comment:
Can you directly explain the benefits of setting a read/write timeout instead of indirectly talking the disadvantages of not setting a timeout?